src/styles/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  width: $calendar-width;
+  margin: 100px auto;
+  padding: 10px;
+
+  flex-wrap: wrap;
+  gap: $gap;
+
+  &__day {
+    display: flex;
+  
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    text-align: center;
+    color: black;
+    
+    background-color: #eee;
```

Comment:
it's good practice move all colors to variables. Please fix all cases if present

Reply:
done